src/containers/App/App.js

```diff
@@ -424,5 +476,5 @@ export default hot(
   connect(
     mapStateToProps,
     mapDispatchToProps
-  )(App)
+  )(injectIntl(App))
```

Comment:
We can't use `injectIntl` here, it has to be on a descendant of the `IntlProvider`.

![image](https://user-images.githubusercontent.com/2829095/83883788-dcf25700-a73b-11ea-9ad4-05bc7c68ac23.png)

The easiest thing to do for now would be to extract the InlineNotification to a function that we can use as a component and wrap that with `injectIntl`.

At the top of App.js add:

```js
const ConfigErrorComponent = ({ intl, loadingConfigError }) => {
  if (!loadingConfigError) {
    return null;
  }

  return (
    <InlineNotification
      kind="error"
      title={intl.formatMessage({
        id: 'dashboard.app.loadingConfigError',
        defaultMessage: 'Error loading configuration'
      })}
      subtitle={getErrorMessage(loadingConfigError)}
      lowContrast
    />
  );
};

const ConfigError = injectIntl(ConfigErrorComponent);
```

Then replace the code that displays the notification with:

```diff
+        <ConfigError loadingConfigError={loadingConfigError} />
-        {loadingConfigError && (
-          <InlineNotification
-            kind="error"
-            title={intl.formatMessage({
-              id: 'dashboard.app.loadingConfigError',
-              defaultMessage: 'Error loading configuration'
-            })}
-            subtitle={getErrorMessage(loadingConfigError)}
-            lowContrast
-          />
-        )}
```

and remove the `injectIntl` on `App`

Reply:
Done, thank you for the fix !